### ISSUE TITLE
I/O CC bitstream generation error

### ISSUE THREAD

**Issue by Kuree** [stevenbell/ultrazynqbuilder#1]
There is no error in synthesis but 2 errors in implementation stage. I'm using Vivado 17.2

Error messages:
```
[DRC NSTD-1] Unspecified I/O Standard: 6 out of 16 logical ports use I/O standard (IOSTANDARD) value 'DEFAULT', instead of a user assigned specific value. This may cause I/O contention or incompatibility with the board power or connectivity affecting performance, signal integrity or in extreme cases cause damage to the device or the components to which it is connected. To correct this violation, specify all I/O standards. This design will fail to generate a bitstream unless all logical ports have a user specified I/O standard value defined. To allow bitstream creation with unspecified I/O standard values (not recommended), use this command: set_property SEVERITY {Warning} [get_drc_checks NSTD-1].  NOTE: When using the Vivado Runs infrastructure (e.g. launch_runs Tcl command), add this command to a .tcl file and add that file as a pre-hook for write_bitstream step for the implementation run. Problem ports: fmc_iic_scl_io, fmc_iic_sda_io, and cam_gpio_tri_o[3:0].
[DRC UCIO-1] Unconstrained Logical Port: 6 out of 16 logical ports have no user assigned specific location constraint (LOC). This may cause I/O contention or incompatibility with the board power or connectivity affecting performance, signal integrity or in extreme cases cause damage to the device or the components to which it is connected. To correct this violation, specify all pin locations. This design will fail to generate a bitstream unless all logical ports have a user specified site LOC constraint defined.  To allow bitstream creation with unspecified pin locations (not recommended), use this command: set_property SEVERITY {Warning} [get_drc_checks UCIO-1].  NOTE: When using the Vivado Runs infrastructure (e.g. launch_runs Tcl command), add this command to a .tcl file and add that file as a pre-hook for write_bitstream step for the implementation run.  Problem ports: fmc_iic_scl_io, fmc_iic_sda_io, and cam_gpio_tri_o[3:0].
```

`hwconfig.all`:
```yaml
 board: iocc
 hw:
 - {name: dma0, outputto: canny, type: dma}
 - name: canny
   outputto: dma0
   path: /nobackup/sebell/work/ultrazed/ip_repo/xilinx_com_hls_hls_target_1_0/
   streams:
   - {depth: 2, name: arg_0, type: input}
   - {depth: 3, name: arg_1, type: output}
   type: hls
   vlnv: xilinx.com:hls:hls_target:1.0
 name: example
 productionsilicon: y
```

**Comment by stevenbell** [comment]
Oh, this is because the I2C port for controlling the camera is still included in the IOCC build, even though the pins aren't there.  I'll put this in an if-block so it only runs on the PCIeCC.  You should be able to work around it by deleting the constraint file in the vivado project.

**Comment by stevenbell** [comment]
Should be fixed in 686b02be6a8323d7c2b1d804b033bd57293f18f0.
